content/en/tracing/connect_logs_and_traces/java.md

```diff
@@ -16,43 +16,22 @@ further_reading:
       tag: 'Blog'
       text: 'Correlate request logs with traces automatically'
 ---
+## Configure Your Logger
 
-## Automatically Inject Trace and Span IDs
-
-Enable injection in the Java tracer's [configuration][1] by adding `-Ddd.logs.injection=true` as a jvm startup argument or through environment variable `DD_LOGS_INJECTION=true`.
-
-If your logs are JSON formated and you are using Logback there is nothing left to do. Otherwise with other logging libraries you need to activate MDC attributes autoinjection into your logs.
-
-If your logs are raw formatted, update your formatter to include `dd.trace_id` and `dd.span_id` in your logger configuration:
-
-{{< tabs >}}
-{{% tab "Log4j2" %}}
+The first step to correlating logs and traces is to [configure your logger][4].  Follow the configuration instructions for your specific logging framework.
```

Comment:
Can we call out here specifically the loggers we support or that you need to configure - something like:
"Follow the config instructions for log4j, log4j2, or logback"